Add notes about RAM limits (#3052)
ArduPilot/ardupilot_wiki: common/source/docs/common-limited-firmware.rst

   -  Parachute
   -  Sprayer

RAM Limitations
===============

> Occasionally, in the above boards, RAM will become limited with enabled options in the firmware also. This is manifested by various symptoms:

Reviewer: ```suggestion
There may be insufficient RAM available in some flight controllers to support all enabled firmware features. This is manifested by various symptoms:
```

Reviewer 2: Should probably be simply, `Some symptoms are`.

(the symptoms would be of the manifestation, but that's implied).

Author: done

> - MAVFtp does not work for fast connections to GCS

Reviewer: ```suggestion
- MAVFtp does not work: parameter fetches to GCS will be slow
```

Reviewer 2: Should be `MAVFTP`  (FTP and MAV both being acronyms).

Author: done
